libs/blocks/icon-block/icon-block.css

```
@@ -88,46 +164,81 @@
   margin-bottom: var(--spacing-s);
 }
 
-.icon-block.vertical .foreground .text {
-  margin-bottom: var(--spacing-xxl);
+.five-up .foreground .text-content {
+  text-align: left;
+  padding-bottom: var(--spacing-xs);
 }
 
-.icon-block.vertical .foreground .icon-area {
-  margin: 0;
-  margin-bottom: var(--spacing-xxs);
+.five-up .foreground .text-content .body-M {
+  font-size: var(--type-body-s-size);
+  line-height: var(--type-body-s-lh);
+}
+
+.icon-block.centered .foreground .text-content .action-area > a:not(.con-button) {
+  display: block;
+  margin-top: var(--spacing-xs);
 }
 
-.icon-block.vertical .foreground .text .action-area>a:not(.con-button) {
-  font-size: 14px;
-  line-height: 21px;
+.icon-block.vertical .foreground .text-content .action-area > a:not(.con-button),
+.icon-block.centered .foreground .text-content .action-area > a:not(.con-button),
+.icon-block.vertical .foreground .text-content p.body-M a:only-child {
+  font-size: var(--type-body-xs-size);
+  line-height: var(--type-body-xs-lh);
 }
 
-.section[class*='-up'] .full-width {
+.icon-block.fullwidth.small .foreground .text-content .action-area > a:not(.con-button) {
+  display: block;
+  margin-top: var(--spacing-s);
+  font-size: var(--type-body-m-size);
+  line-height: var(--type-body-m-lh);
+}
+
+.two-up .icon-block.fullwidth,
+.three-up .icon-block.fullwidth,
+.four-up .icon-block.fullwidth,
+.five-up .icon-block.fullwidth {
   grid-column: 1 /-1;
 }
 
-@media screen and (min-width: 600px) {
+.icon-block.fullwidth .foreground,
+.icon-block.bio .foreground {
+  width: var(--grid-container-width);
+  margin: 0 auto;
+}
+
+@media screen and (min-width: 600px) and (max-width: 1200px) {
   .icon-block .foreground .icon-area {
     max-width: 234px;
   }
 
-  .icon-block.vertical .foreground {
-    gap: 32px;
+  .five-up .icon-block .foreground .text-content {
+    text-align: center;
+  }
+
+  .five-up .icon-block .foreground .text-content .icon-area {
+    text-align: center;
   }
```

Can consolidate these
```suggestion
  .five-up .icon-block .foreground .text-content,
  .five-up .icon-block .foreground .text-content .icon-area {
    text-align: center;
  }
```